cmd/repo_install.go

```diff
@@ -24,17 +24,14 @@ import (
 
 func init() {
 	repoInstallCmd := &cobra.Command{
-		Use:   "install",
+		Use:   "install [artifact...]",
 		Short: "Install an artifact",
 		Long:  `Install an artifact`,
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
I'm not sure I want to force a number of args...

Reply:
Does it make sense to run this command without any argument?